ohai.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "ipaddress"
   s.add_dependency "wmi-lite", "~> 1.0"
   s.add_dependency "ffi", "~> 1.9"
+  s.add_dependency "chef-config", "~> 12.4.1"
```

Comment:
I could see an argument actually for chef-config "~> 12.4" since I may want to upgrade ohai or chef entirely independently of each other provided that they stay on the same major version number.  We should ideally be able to still replicate omnibus builds since we should pin the version of the chef and ohai gems and the chef-config version will then just get depsolved.  I don't know about the implications for gem-installed chef and ohai though (although clearly version pinning there would also solve this problem).

So I think I'm leaning towards a much less restrictive pin here.